src/services/mentees.js

```diff
@@ -1313,7 +1315,7 @@ module.exports = class MenteesHelper {
 					 */
 					filter =
 						additionalFilter +
-						`AND ( (${userPolicyDetails.organization_id} = ANY("visible_to_organizations") AND "external_session_visibility" != 'CURRENT')`
+						`AND ( (${userPolicyDetails.organization_id} = ANY("visible_to_organizations") AND "external_mentee_visibility_policy" != 'CURRENT')`
```

Comment:
It should be mentee_visibility_policy , not external_mentee_visibility_policy. @Prajwal17Tunerlabs